scripts/helpers/eosio.sh

```diff
@@ -315,26 +315,26 @@ function ensure-libpq-and-libpqxx() {
         if [[ $NAME == "Amazon Linux" ]]; then
             #install libpq
             if [ ! -d /usr/include/libpq ]; then
-                amazon-linux-extras enable postgresql11 && \
-                    yum install -y libpq-devel 
+                sudo amazon-linux-extras enable postgresql11 && \
+                    sudo yum install -y libpq-devel
             fi
             EXTRA_CMAKE_FLAGS="-DPostgreSQL_TYPE_INCLUDE_DIR=/usr/include/libpq"
         elif [[ $NAME == "CentOS Linux" ]]; then
             #install libpq
             if [ ! -d /usr/pgsql-13 ]; then
                 CENTOS_VERSION=$(rpm -E %{rhel})
-                yum install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-$CENTOS_VERSION-x86_64/pgdg-redhat-repo-latest.noarch.rpm
-                [ $CENTOS_VERSION -lt 8 ] || dnf -qy module disable postgresql
-                yum install -y postgresql13-devel 
+                sudo yum install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-$CENTOS_VERSION-x86_64/pgdg-redhat-repo-latest.noarch.rpm
+                [ $CENTOS_VERSION -lt 8 ] || sudo dnf -qy module disable postgresql
+                sudo yum install -y postgresql13-devel
             fi
             export PostgreSQL_ROOT=/usr/pgsql-13   
             export PKG_CONFIG_PATH=/usr/pgsql-13/lib/pkgconfig
         elif [[ $NAME == "Ubuntu" ]]; then
             # install libpq
             if [ ! -d /usr/include/postgresql ]; then 
-                echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list && \
-                    curl -sL https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - && \
-                    apt-get update && apt-get -y install libpq-dev  
+                sudo echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > sudo /etc/apt/sources.list.d/pgdg.list && \
```

Comment:
Does this work properly? The redirection w/ sudo usually requires more care.

Reply:
Yes, Per my test it can pass the build, however, the line 'echo "deb http://apt... ' more like did nothing after run.

Reply:
Won't
```
sudo echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > sudo /etc/apt/sources.list.d/pgdg.list
```
create a file named `sudo`?

Reply:
no, create nothing. just like no echo called.

Reply:
`sudo echo "deb http://apt.postgresql.org/pub/repos/apt -pgdg main" > sudo hello`
Creates file `sudo` with contents:
```
bash-3.2$ cat sudo
deb http://apt.postgresql.org/pub/repos/apt -pgdg main hello
```

Reply:
Funny it actual includes "hello", did not expect that. But pretty clear this is not what you want.

Reply:
Maybe, it didn't check carefully in currently directly, I checked /etc/apt/sources.list.d nothing changed, even no pgdg.list there.